packages/noya-state/src/selectors/layerSelectors.ts

```diff
@@ -70,6 +70,21 @@ export const getSelectedLayers = (
   ) as PageLayer[];
 };
 
+export const getSelectedLayersWithTopLevelCoordinates = (
+  state: Draft<ApplicationState>,
+): PageLayer[] => {
+  const page = getCurrentPage(state);
+
+  const selectedLayers = Layers.findAll(page, (layer) =>
+    state.selectedLayerIds.includes(layer.do_objectID),
+  ) as PageLayer[];
+
+  return selectedLayers.map((layer) => {
+    resizeLayerFrame(layer, page.frame);
```

Comment:
Does this work for nested layers? What about rotated layers? I would think we'd need to do this using transforms, but if this works then it's fine.

Since `resizeLayerFrame` mutates the layer object, we would probably want to wrap things in a `produce` to first clone the layers.

Reply:
ohh I think not

Reply:
I don't even remember why I was using the `resizeLayerFrame` 😆 I just noticed it's not even been called.

Reply:
Okey it does work for groups 😆 It works. I only need to delete the resize layer that I don't even need. I think this how I was trying to do it before and left it like this. I just need the selected layer